code/presentation/mb-client/src/navigation/home-navigator.tsx

```diff
@@ -0,0 +1,68 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import * as React from 'react';
+
+import { supabase } from '@/core/supabase'; // Import supabase instance
+import { HomeScreen } from '@/screens/home';
+import AudioScreen from '@/screens/home/audio-screen';
+import FindMasjid from '@/screens/home/find-masjid';
+import MasjidInfo from '@/screens/home/masjid-info';
+import MasjidScreen from '@/screens/home/masjid-screen';
+import NamazTimingsScreen from '@/screens/home/namaz-timings';
+import VideoScreen from '@/screens/home/video-screen';
```

Comment:
Can we use screen within screen? If not, make components which are not actually a screen into a component.

Reply:
couldn't understand. can u elaborate